docs/guides/streaming/overview.mdx

```diff
@@ -18,21 +18,21 @@ pipeline:
 ![Complex streaming pipeline](https://media.graphassets.com/4ZWHyLpgTESu0rB1sXnP)
 
 ## Supported sources
-- [Amazon SQS](sources/amazon-sqs)
```

Comment:
actually, i think the relative paths work. https://docs.mage.ai/guides/streaming/overview#supported-sources